logistics_project/apps/malawi/nag.py

```diff
@@ -129,9 +130,9 @@ def _get_hsas_ready_for_nag(warning_time, extra_filter_params={}):
                                         nag_type=Reports.REC,
                                         **extra_filter_params).count():
 
-                if not ProductReport.objects.filter(report_type__code=Reports.REC,
-                                                 report_date__range=[req.responded_on,
-                                                                     now]).exists():
+                if not ProductReport.objects.filter(supply_point=req.supply_point,
```

Comment:
It looks like the original intention of this was to send a reminder to all HSAs with approved StockRequests that don't yet have a NagRecord and haven't submitted a receipt ProductReport since the StockRequest was approved. But it wasn't filtering on supply point in the ProductReport filter, so it was counting any receipt in the date range as a receipt for this HSA. This seemed wrong so I added supply_point to the filter here. @czue fyi